examples/platforms/posix/ncp/hdlc.cpp

```diff
@@ -0,0 +1 @@
+../../../../src/ncp/hdlc.cpp
```

Comment:
As this works perfectly on Linux, the Windows OS shows exactly what Github shows.

What about to move hdlc.cpp/hpp to utils folder and make use of it both in NCP and RADIO_NCP?

Reply:
This is not going to support windows, and windows has its own way. In fact, I don't have a better solution for references a single source file in multiple targets.  Autotools doesn't support this well.

Reply:
Ok i wasn't aware about this autotools limitation. I understand that the output file of the compilation will be used in posix only, i was talking more about situation where development for Posix is done on Windows. Saying that, i am fine with current solution, if autotools does not support it 👍